database/seedPostgres.js

```diff
@@ -0,0 +1,86 @@
+const faker = require('faker');
+const fs = require('fs');
```

Comment:
if I don't miss anything, I can't find any line, which ***fs*** is being used. 
Maybe you can remove the unused module on next PR.

Reply:
Thanks! I thought maybe I would use streams to get the data into the db but then I changed my plan.